Content.Shared/LieDown/LyingDownComponent.cs

```diff
@@ -0,0 +1,25 @@
+using Content.Shared.Actions;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Shared.LieDown;
+
+/// <summary>
+///     Makes the target to lie down.
+/// </summary>
+[Access(typeof(SharedLieDownSystem))]
+[RegisterComponent, NetworkedComponent()]
+public sealed partial class LyingDownComponent : Component
+{
+    /// <summary>
+    ///     The action to lie down or stand up.
+    /// </summary>
+    [DataField("make-to-stand-up-action", customTypeSerializer: typeof(PrototypeIdSerializer<EntityPrototype>))]
+    public string? MakeToStandUpAction = "action-name-make-standup";
```

Comment:
```suggestion
    public EntProtoId? MakeToStandUpAction = "action-name-make-standup";
```
И нафиг удаляете кастом сериалайзеры